src/pages/event/eventDetail/eventDetail.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { FunctionComponent } from "react";
+import { useLocation, useParams } from "react-router-dom";
+import { EventProps } from "../../../components/UI/ResourceEvent";
+import { events } from "../event";
+import { Helmet } from "react-helmet";
+import { Link } from "react-router-dom";
+import { ChevronLeft, ExternalLink, PlayCircle, Download } from "react-feather";
+import { Page } from "../../../components/Layout/Page";
+import { isFuture, format } from "date-fns";
+import { formatTime } from "../../../common/utils/dateTime";
+import { LinkButton } from "@govtechsg/tradetrust-ui-components";
+import ReactMarkdown from "react-markdown";
+
+export const EventPageDetail: FunctionComponent = () => {
+  const locationPath = useLocation();
+  const params: { slug: string } = useParams();
+  const detail: EventProps = events.find((eventDetail) => eventDetail.slug === params.slug);
+
+  const {
+    title,
+    thumbnail,
+    blurb,
+    link,
+    date,
+    timeStart,
+    timeEnd,
+    videoLink,
+    slides,
+    registerLink,
+    location,
+    downloadableMediaContent1,
+    downloadableMediaContent2,
+    downloadableMediaContent3,
+    eventDetails,
+  } = detail.attributes;
+  const downloadableContents = [downloadableMediaContent1, downloadableMediaContent2, downloadableMediaContent3];
+
+  return (
+    <>
+      <Helmet>
+        <meta
+          property="description"
+          content="Check out our events and browse through our latest news and official statements."
+        />
+        <meta
+          property="og:description"
+          content="Check out our events and browse through our latest news and official statements."
+        />
+        <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
+        <meta property="og:url" content={`${window.location.origin}${locationPath.pathname}`} />
+        <title>TradeTrust - {title}</title>
+      </Helmet>
+      <Page title="Event">
+        <div className="flex my-4">
+          <div className="w-auto">
+            <Link to="/event" className="text-gray-800 flex flex-nowrap items-center">
+              <ChevronLeft />
+              <span>Back</span>
+            </Link>
+          </div>
+        </div>
+        <div className="w-full px-4 lg:w-9/12">
+          <div className="bg-white text-gray-600 shadow-lg rounded-lg p-8">
+            <h2 className="font-medium lg:font-bold text-2xl lg:text-4xl mb-6 lg:mb-10">{title}</h2>
```

Comment:
in smaller screen view, looks odd that `h2` have smaller font size (24px) compared to wysiwyg's `h3` (26px). unless there is a mobile mock provided, otherwise would suggest to: 

```suggestion
            <h2 className="font-medium lg:font-bold text-4xl mb-6 lg:mb-10">{title}</h2>
```

Reply:
ok understand, updated